src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline (String description, String by){
+        super(description, "D");
```

Comment:
I was thinking perhaps you can consider defining a variable that corresponds to "D"? It might be better to not use a magic string like "D" that does not explain anything about its meaning